BuildAddOn.py

```diff
@@ -12,12 +12,13 @@
 
 def ParseArguments ():
     parser = argparse.ArgumentParser ()
-    parser.add_argument ('-c', '--configFile', dest = 'configFile', required = True, help = 'JSON Configuration file')
+    parser.add_argument ('-c', '--languageConfig', dest = 'languageConfig', required = True, help = 'JSON language configuration file')
     parser.add_argument ('-v', '--acVersion', dest = 'acVersion', nargs = '+', type = str, required = False, help = 'Archicad version number list. Ex: 26 27')
-    parser.add_argument ('-l', '--language', dest = 'language', nargs = '+', type = str, required = False, help = 'Add-On language code list. Ex: INT GER. Specify ALL for all languages in the configfile.' )
+    parser.add_argument ('-l', '--language', dest = 'language', nargs = '+', type = str, required = False, help = 'Add-On language code list. Ex: INT GER. Specify ALL for all languages in the languageConfig file.' )

```

Comment:
I'm not sure that this `ALL` behavior will be straightforward for users. What will these parameter sets mean?

```
--languageConfig languages.json --language ALL
--languageConfig languages.json --language GER
--language ALL
```

I think we should choose one, either the config file or the language parameter. I think keeping only the language parameter can be the most straightforward with `INT` as default value, and eliminating `ALL` handling.